server/local/studentdashboard-LOCALHOST.yml

```diff
@@ -55,7 +59,7 @@ latte_admin_group: TL-Latte-admin-test
 ## from local files.  Configuration is documented below.
 
 ### file based data provider value:
-data_provider_file_directory: server/test-files/courses
+data_provider_file_directory: server/test-files
```

Comment:
The  server/test-files/courses/ststvii.json is still inside courses folder. Is it OK to omit "courses" here in the data_provider_file_directory setting?